Genres/Qene.xml

```diff
@@ -33,9 +33,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <p>This keyword is used to classify genres of texts appearing in the manuscripts (for the types of additiones, a specific keyword <ref type="authFile"
                     corresp="PoemQene"
                     >PoemQene</ref> is additionally used.</p>
-                <p>Despite wide-spread opinion that qǝne is a poetic improvisation which is not written down, manuscripts contain unexpectedly many qǝne-poems, usually, but not only as additional notes. 
-                    Frequently it is clearly stated the the poem is qǝne, and even the type is indicated. But no one tried to check if these poems formally correspond to what we know today as qǝne-forms. </p>
-                <p>Wherever possible, we mark qǝne-forms with this specific keyword; when it is indicated, we also specify the specific modes, see <ref type="authFile" corresp="EtanaMogar">ʿəṭāna mogar</ref>,  <ref type="authFile" corresp="Mawaddes">mawaddəs</ref></p>
+                <p>Despite wide-spread opinion that qǝne is a poetic improvisation which is not written down, manuscripts contain unexpectedly many of them in additional notes or in collections of qǝne-poems. 
+                    Frequently it is clearly stated the poem is qǝne, and even the type is indicated.</p>
+                <p>Wherever possible, we define qǝne-forms according to the classification 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl> and mark it with this specific keyword and
+                    Cf. 
+                    <ref type="authFile" corresp="EtanaMogar"/>,  
```

Comment:
I guess there should be a comma after each of these keywords.

Maybe they could be sorted alphabetically?

Reply:
Possible; but Guidi arranged them according to the length (number of verses, starting from the guba'e qana of two verses). As step two, we can include short info on this subject, to each of the type.

Reply:
If it's made clear that this is the logic behind the sorting, it would also work well